src/app/Main/Body/QueryPage/Subheader/SubHeader-DefaultQueries.component.tsx

```diff
@@ -0,0 +1,64 @@
+/** @jsx jsx */
+import { css, jsx, SerializedStyles } from "@emotion/core";
+import { Button } from "@mui/material";
+import QueryStatsIcon from "@mui/icons-material/QueryStats";
+import {
+    onPrimaryColor,
+    secondaryColor,
+} from "../../../../Shared/Style/Style-MainTheme";
+
+const buttonLabelStyle = (isOpen: boolean): SerializedStyles => css`
+    display: flex;
+    align-items: center;
+    &:hover {
+        color: ${isOpen ? "none" : secondaryColor};
+    }
+`;
+
+const downloadButtonStyle = css`
+    margin-right: 1em;
+    padding: 2px 4px;
+    line-height: 0px;
+    text-transform: none;
+    color: ${onPrimaryColor};
+    a {
+        padding: 0px;
+        font-size: 1rem;
+        line-height: 1rem;
+        font: 400 14px/20px Arial, sans-serif;
+    }
+`;
+
+/**
+ * @desc Returns the export button inside the header.
+ * @param props
+ * @returns {JSX.Element} - export button component
+ */
+export function SubHeaderDefaultQueriesButtonComponent(props: {
```

Comment:
Might be useful to use a common subHeaderButtonComponent

Reply:
Done